packages/perseus/src/perseus-types.ts

```diff
@@ -909,6 +909,79 @@ export type PerseusGraphTypeRay = {
     startCoords?: CollinearTuple;
 } & PerseusGraphTypeCommon;
 
+type AngleGraphCorrect = {
+    type: "angle";
+    // Whether to show the angle measurements.  default: false
+    allowReflexAngles?: boolean;
+    match?: "congruent";
+    coords: [Coord, Coord, Coord];
+};
+
+type CircleGraphCorrect = {
+    type: "circle";
+    center: Coord;
+    radius: number;
+};
+
+type LinearGraphCorrect = {
+    type: "linear";
+    coords: CollinearTuple;
+};
+
+type LinearSystemGraphCorrect = {
+    type: "linear-system";
+    coords: [CollinearTuple, CollinearTuple];
+};
+
+type NoneGraphCorrect = {
+    type: "none";
+};
+
+type PointGraphCorrect = {
+    type: "point";
+    coords: ReadonlyArray<Coord>;
+};
+
+type PolygonGraphCorrect = {
+    type: "polygon";
+    match?: "similar" | "congruent" | "approx";
+    coords: ReadonlyArray<Coord>;
+};
+
+type QuadraticGraphCorrect = {
+    type: "quadratic";
+    coords: [Coord, Coord, Coord];
+};
+
+type SegmentGraphCorrect = {
+    type: "segment";
+    // Expects a list of Coord tuples. Length should match the `numSegments` value.
```

Comment:
There's no way for the learner to add or remove segments, so I don't know why we'd check `numSegments`. I'd just as soon delete this comment, since it's the kind of thing that's likely to go out of date (it refers to logic in another part of the codebase).

Reply:
Deleted :)